README.md

```diff
@@ -35,19 +35,21 @@ Function overview:
 
 ### addTimer
 
-Instead of calling InternalTimer you add the Timer with this command:
+Instead of calling FHEMs `InternalTimer()` you can add a timer with `addTimer()`:
 
-**addTimer($name,$timestamp, $coderef, $arg, $waitIfInitNotDone);**
+**addTimer($name,$timestamp, $function_ref, $arg, $waitIfInitNotDone);**
 
 Example:
-`FHEM::Core::Timer::Helper::addTimer($hash{NAME}, time(), \&someSub,"paramsGoesHere",0 );`
-| parameter     | required | description  |
-| ------------- |:-------------:| -----:|
-| $name                 |mandatory   |A name or identifier to specify for this Timer|
-| $timestamp            |mandatory   |Unix timestamp when the timer should run|
-| $functionRef          |mandatory   |Is a codref to some code to run / call after $timestamp is reache|
-| $arg                  |optional    |Arguments you pass to the coderef, default = {} |
-| $waitIfInitNotDone    |operional   |Blocks FHEM until $timestamp is reached, use it only if really needed|
+
+`FHEM::Core::Timer::Helper::addTimer( $hash{NAME}, time(), \&do_something, { argument => q{value} }, 0 );`
+
+| Parameter             | Required?   | Description  |
+| :---------------------|:----------- | :----------  |
+| `$name`               | mandatory   | A label for this timer|
+| `$timestamp`          | mandatory   | Unix timestamp when the timer should run|
+| `$function_ref`       | mandatory   | Is a codref to some code to run / call if `$timestamp` is reached|
+| `$arg`                | optional    | Hash with arguments, passed to the `$function_ref`, default = `{}`(empty hash) |
```

Comment:
Hi,

Danke für die Überarbeitung :)
An dieser Stelle ist deine Annahme nicht ganz richtig.

Als $arg kann prinzipiell alles übergeben werden, was als eine Variable erkannt wied. Default ist ein leerer String q{}

Reply:
Ja, hab's mir angeguckt und in 1f4f49442c8344dc8e6ed9bd94732334a75a25eb gefixt.